src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -99,6 +101,56 @@ public void setGuiSettings(GuiSettings guiSettings) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public Path getLinkyTimeFilePath() {
+            return null;
+        }
+
+        @Override
+        public void setLinkyTimeFilePath(Path linkyTimeFilePath) {
+
+        }
+
+        @Override
+        public void setLinkyTime(ReadOnlyLinkyTime linkyTime) {
+
+        }
```

Comment:
```suggestion
        @Override
        public void setLinkyTime(ReadOnlyLinkyTime linkyTime) {
            throw new AssertionError("This method should not be called.");
        }
```
Should add the assertion error.
Same for the other methods in this file.